### DISCUSSION TITLE
Report website errors to a separate repo

### DISCUSSION THREAD

**Discussion by SoftCreatR** [ruffle-rs/ruffle#3423] [Ideas]
For a significantly better overview, bugs reported via ruffle should end up in a separate repository (Those titled "Error on...", which are more than 1/3 of all issues reported so far). The problem here is that GitHub does not support negating search filters, otherwise you could just make use of this.

Of course, it's also possible that this does work somehow. In that case, my suggestion can simply be ignored.

**Comment by Poopooracoocoo** [comment]
@Herschel thoughts? the repo has hit 1000 issues 😱

**Comment by danielhjacobs** [comment]
While this may be a good idea, Github actually **does** support negating search filters. See [this link](https://github.com/ruffle-rs/ruffle/issues?q=is:open+is:issue+NOT+"Error+on"+in:title+AND+NOT+%22ActionScript+Tracking:%22+in:title+-label:panic+-label:error-report).
